matlab/lpo_mex.cpp

```diff
@@ -383,11 +324,15 @@ static void Proposal_propose( MEX_ARGS ) {
         mexErrMsgTxt("Expected a single output argument");
         return;
     }
-    std::shared_ptr<Proposal> p = mat2Ptr<Proposal>( prhs[0] );
+    std::shared_ptr<LPO> p = mat2Ptr<LPO>( prhs[0] );
     std::shared_ptr<ImageOverSegmentation> os = mat2Ptr<ImageOverSegmentation>( prhs[1] );
-    
-    RMatrixXb r = p->propose( *os );
-    
+
+    p->load( model );
+
+    std::vector<Proposals> s = p->propose( *os );
+
+    RMatrixXb r = s[0].p;
```

Comment:
I'm not at all familiar with matlab, but we should find a way to return all proposals, and both the p (binary mask of superpixels in proposals) and s (superpixel segmentation) of each set of proposals. A list of structs might work.